ocaml/lambda/translprim.ml

```diff
@@ -60,15 +61,15 @@ type comparison =
   | Compare
 
 type comparison_kind =
-  | Compare_generic
-  | Compare_ints
+  | Compare_generic of { signed : bool }
```

Comment:
Polymorphic comparison probably shouldn't have an unsigned option, right?

Reply:
Right. They all start as `Compare_generic` in `lookup_primitive`. I will try to move this `bool` to the `Comparison` constructor of `prim`, it'll be cleaner. Thanks!